pkg/plugin/state.go

```diff
@@ -669,15 +669,17 @@ func (e *AutoscaleEnforcer) handlePodDeletion(podName util.NamespacedName) {
 	klog.Infof(fmtString, podName, pod.node.name, cpuVerdict, memVerdict)
 }
 
-func (e *AutoscaleEnforcer) handleUpdatedScalingBounds(vm *api.VmInfo, podName string) {
+func (e *AutoscaleEnforcer) handleUpdatedScalingBounds(vm *api.VmInfo, unqualifiedPodName string) {
 	e.state.lock.Lock()
 	defer e.state.lock.Unlock()
 
-	klog.Infof("Handling updated scaling bounds for VM pod %s:%s", vm.Namespace, podName)
+	podName := util.NamespacedName{Namespace: vm.Namespace, Name: unqualifiedPodName}
 
-	pod, ok := e.state.podMap[util.NamespacedName{Namespace: vm.Namespace, Name: podName}]
+	klog.Infof("Handling updated scaling bounds for VM pod %v", podName)
+
+	pod, ok := e.state.podMap[podName]
 	if !ok {
-		klog.Errorf("[autoscale-enforcer] cannot find VM pod %s:%s to update scaling bounds", vm.Namespace, podName)
+		klog.Errorf("[autoscale-enforcer] cannot find VM pod %v to update scaling bounds", podName)
```

Comment:
isn't this stil a string/stringer?

Reply:
discussed IRL - not a stringer because it's implemented with Format